GpsDataCapturer/mobile/src/main/java/com/google/gpsdatacapturer/MobileGpsMainActivity.java

```diff
@@ -196,6 +208,29 @@ public void onServiceConnected(ComponentName name, IBinder service) {
             //get the gpsDataCaptureService
             gpsDataCaptureService =
                     ((GpsDataCaptureService.GpsDataCaptureBinder) service).getService();
+
+            //start capture via intent
+            if(adbIntent != null) {
```

Comment:
You may want to compare it with action string "com.google.gpsdatacapturer.START_CAPTURE"

Reply:
Yes, I did it on line 228.

Reply:
Please move the comparison here